client/src/Views/Landing/gmailAuth.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Contains functions involved in checking and getting gmail authorisation for user
+ */
+
+const axios = require("axios");
```

Comment:
You should be using `import` here as well.

Also, I forgot to mention this, but I don't think this file should be in `src/Views`. I think Shums mentioned having a separate folder like `src/Utils` for files like these, and since I also need to do the API calls for /login and /register, I was wondering if you think we should have them all in one file or in separate files?

Reply:
Thanks, I'll change it.

I agree. I'll move it to /src/Utils. Given Shums recommended to include them all in the same file, let's do that. I'll rename the file. Is there a standard naming convention? api.js?